python/pyspark/sql/column.py

```diff
@@ -720,8 +817,20 @@ def isin(self, *cols: Any) -> "Column":
 
         .. versionadded:: 1.5.0
 
+        Parameters
+        ----------
+        cols
+            The result will only be true at a location if all the labels match in the Column.
```

Comment:
Shouldn't it be "any" instead of "all"? Also "value" might be a better wording than "label".

Reply:
Thanks. Better match the method function.